Classes/Request/YLPGeoCoordinate.m

```diff
@@ -31,15 +31,15 @@ - (instancetype)initWithLatitude:(double)latitude
 - (NSString *)description {
     NSString *baseString = [self latLongString];
     
-    if (self.accuracy) {
+    if (self.accuracy != 0) {
         baseString = [self accuracyDescription:baseString];
     }
     
-    if (self.altitude) {
+    if (self.altitude != 0) {
         baseString = [self altitudeDescription:baseString];
     }
     
-    if (self.altitudeAccuracy) {
+    if (self.altitudeAccuracy != 0) {
         baseString = [self altitudeAccuracyDescription:baseString];
```

Comment:
@SSheldon I was actually under the assumption that the API returned error responses when 0 values were passed in so I was only adding them into the output string if they were provided. But I just did some searches with 0's for accuracy/altitude/altitudeAccuracy and they all worked fine, so let's just get rid of these checks, and put in the default 0 if they aren't provided.